build.psm1

```diff
@@ -2361,29 +2361,7 @@ function Start-CrossGen {
         "Microsoft.ApplicationInsights.dll"
     )
 
-    # Common PowerShell libraries to crossgen
-    $psCoreAssemblyList = @(
-        "pwsh.dll",
-        "Microsoft.PowerShell.Commands.Utility.dll",
-        "Microsoft.PowerShell.Commands.Management.dll",
-        "Microsoft.PowerShell.Security.dll",
-        "Microsoft.PowerShell.ConsoleHost.dll",
-        "System.Management.Automation.dll"
-    )
-
-    # Add Windows specific libraries
-    if ($environment.IsWindows) {
-        $psCoreAssemblyList += @(
-            "Microsoft.PowerShell.CoreCLR.Eventing.dll",
-            "Microsoft.WSMan.Management.dll",
-            "Microsoft.WSMan.Runtime.dll",
-            "Microsoft.PowerShell.Commands.Diagnostics.dll",
-            "Microsoft.PowerShell.GraphicalHost.dll",
-            "Microsoft.Management.Infrastructure.CimCmdlets.dll"
-        )
-    }
-
-    $fullAssemblyList = $commonAssembliesForAddType + $psCoreAssemblyList
+    $fullAssemblyList = $commonAssembliesForAddType
```

Comment:
It seems we should disable Start-CrossGen function.

Reply:
We should probably disable `Start-CrossGen` until we can verify PublishReadyToRun is a good replacement

Reply:
Done.

Reply:
CrossGen and ReadyToRun are the same thing...

Reply:
We can use `set COMPlus_ReadyToRunLogFile=log.txt` from issue I referenced above to check dll-s. Start-CrossGen does not create R2R dll-s.

Reply:
@iSazonov crossgen with a switch **is** what is used to generate readytorun binaries in the dotnet project

Reply:
And I switched us to that method, but per DotNet we need to use the csproj if we want our symbols to work:  https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L2217

Reply:
But can your explain why crossgen should be disabled on the rest of the binaries?

Reply:
I forgot that we also crossgen assemblies not part of the build that we include in the package.  We should probably still crossgen those.

Reply:
This make sense for third-party assemblies. For PSReadline and MI we could move to R2R format.

> @iSazonov crossgen with a switch is what is used to generate readytorun binaries in the dotnet project

With `set COMPlus_ReadyToRunLogFile=log.txt` logs show that after Start-CrossGen our dll-s are not initialized as R2R but with PublishReadyToRun are initialized.